ffmpeg-2404-sdk/snapcraft.yaml

```diff
@@ -480,5 +480,5 @@ parts:
     after: [deps]
     override-prime: |
       craftctl default
-      sed -i '/usr\/lib\/x86_64-linux-gnu\/libvulkan.so.1\*/d' ${CRAFT_PART_SRC}/lists/*.list
+      sed -i '/usr\/lib\/${CRAFT_ARCH_TRIPLET_BUILD_FOR}\/libvulkan.so.1\*/d' ${CRAFT_PART_SRC}/lists/*.list
```

Comment:
Best use some other sed delimiters so you don't have to escape. 

Also only mangle the relevant file.

```suggestion
      sed -i '|usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}/libvulkan.so.1\*|d' ${CRAFT_PART_SRC}/lists/{CRAFT_ARCH_BUILD_FOR}.list
```

Reply:
We're removing the complete line here. The `/` is not a delimiter, it belongs to the line itself, which will be removed. Also the list file names are `mesa-2404.amd64.list` and similar, so only `${CRAFT_ARCH_BUILD_FOR}` will not work. Rather, I think keeping it as `*` is a better option.

Reply:
The initial and final `/` are delimiters (`/foo\/bar/d` == `|foo/bar|d`), it's just easier to read without all the `\/`.

`*.${CRAFT_ARCH_BUILD_FOR}.list` would work, too. Anyway, your call. The approach is sane in general.

Reply:
Thanks @brlin-tw for the sleuthing. Didn't realize `sed` wants the initial backslash - since it's not the case for `s|foo|bar|`…